src/logger/luaLoggerBinding.c

```diff
@@ -272,6 +281,98 @@ int Lua_GetButton(lua_State *L)
     return 1;
 }
 
+/**
+ * Initializes the specified serial port
+ * Lua Params:
+ * port - the serial port to initialize
+ *        (SERIAL_USB, SERIAL_GPS, SERIAL_TELEMETRY, SERIAL_WIRELESS, SERIAL_AUX) (defaults to SERIAL_AUX)
+ * baud - Baud Rate (defaults to 115200)
+ * bits - Number of bit in the message (8 or 7) (defaults to 8)
+ * parity - (1 = Even Parity, 2 = Odd Parity, 0 = No Parity) (defaults to No Parity)
+ * stopBits - number of stop bits (1 or 2) (defaults to 1)
+ *
+ * Lua Returns:
+ * no return values (nil)
+ */
+int Lua_InitSerial(lua_State *L)
```

Comment:
all of the lua bindings aren't static, so we're consistent with the existing pattern